_pages/home/updating-to-version-2211.35/migration.md

```diff
@@ -66,7 +66,22 @@ The update to Spartacus 2211.36 is mostly focused on updating the framework to A
 
 **Note:** You must start with a version 2211.32.1 Spartacus app to be able to update to version 2211.36.
 
-1. To update to version 2211.36 of Spartacus, run the following command in the workspace of your Angular application:
+1. In `package.json`, update the version to 2211.36.1 for all Spartacus dependencies except `@spartacus/schematics`.
```

Comment:
In other docs for "Update release 2211.xy" (e.g. 2211.32) we used this phrase. I'm suggesting it for consistency:

> In package.json, set your @spartacus libraries to “~2211.32.1".

... + add "except `@spartacus/schematics`.

```suggestion
1. In package.json, set your @spartacus libraries to “~2211.36.1", except `@spartacus/schematics` (schematics should stay on `~2211.32.1`
```

Reply:
I updated the develop branch with this suggestion. I didn't include the final text in parentheses because I explicitly mention this in the following step, include a note dedicated to this detail 🙂

Reply:
Since the PR is already merged. Sorry for merging before you could include additional feedback!